samples/moby-epub3-exploded/OPS/package.opf

```diff
@@ -15,6 +15,7 @@
     <dc:rights>This work is shared with the public using the Attribution-ShareAlike 3.0 Unported (CC BY-SA 3.0) license.</dc:rights>        
     <link rel="cc:license" href="http://creativecommons.org/licenses/by-sa/3.0/"/>
     <meta property="cc:attributionURL">http://code.google.com/p/epub-samples/</meta>
+    <meta property="rendition:layout">reflowable</meta>

```

Comment:
Samples: https://github.com/IDPF/epub3-samples/blob/main/30/page-blanche-bitmaps-in-spine/EPUB/package.opf